browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -21534,8 +21534,43 @@ index 46388b47a0f433fbdae3874a958fb2d207916c45..d42933e66178ff007a5008dc807483c4
          ruby \
          xfonts-utils"
  
+diff --git a/Tools/jhbuild/jhbuild-minimal.modules b/Tools/jhbuild/jhbuild-minimal.modules
+index 545b04743043..dc7ee36b969f 100644
+--- a/Tools/jhbuild/jhbuild-minimal.modules
++++ b/Tools/jhbuild/jhbuild-minimal.modules
+@@ -8,6 +8,7 @@
+       <dep package="wpebackend-fdo"/>
+       <dep package="libsoup"/>
+       <dep package="manette"/>
++      <dep package="libvpx"/>
```

Comment:
Should we get this upstream?

Reply:
i fully intend to, but for now i wanted to get it here since you need it the most right now